iterative/utils/analytics.go

```diff
@@ -212,54 +254,29 @@ func UserId() (string, error) {
 
 	if os.IsNotExist(errorNew) {
 		if !os.IsNotExist(errorOld) {
```

Comment:
[New code should use `errors.Is` instead](https://pkg.go.dev/os#IsNotExist) 💅🏼 
```suggestion
	if errors.Is(errorNew, fs.ErrNotExist) {
		if !errors.Is(errorOld, fs.ErrNotExist) {
```